src/components/user/Team.jsx

```diff
@@ -1,46 +1,144 @@
+import Button from "../Button";
 import Input from "../Input";
+import { useEffect, useState } from "react";
+import axios from "axios";
+import toast from "react-hot-toast";
+import Loading from "../Loading";
+const Team = ({ user, setUser }) => {
+  const [team, setTeam] = useState(null);
+  const [teamID, setTeamID] = useState({});
+  const [edit, setEdit] = useState(false);
+  const handleLeave = () => {
+    axios.delete("/api/members").then(() => {
+      toast("✅ Successfully leave team!");
+      setTeam(null);
+      setUser({ ...user, team: null });
+      setTeamID({});
+    });
+  };
+  const handleJoin = () => {
+    axios
+      .put("/api/members", { team: teamID.team })
+      .then(() => {
+        toast("✅ Successfully join the team!");
+        setUser({ ...user, team: teamID.team });
+      })
+      .catch(({ response }) => {
+        if (response.data.message === "Excceed 4 People Limit")
+          toast("❌ Excceed 4 People Limit");
+        else if (response.data.message === "Invalid Team ID")
+          toast("❌ Invalid Team ID");
+        else toast("❌ Internal Server Error");
+      });
+  };
+  const handleCreate = () => {
+    axios.post("/api/team").then((response) => {
+      setTeam({
+        name: "No Team Name",
```

Comment:
there should be a reset variable that has these values, and then u just pass in reset into team